bindings/python/src/session.cpp

```diff
@@ -728,20 +728,20 @@ namespace
         sig = sign.bytes;
     }
 
-    void dht_put_mutable_item(lt::session& ses, std::string private_key, std::string public_key,
-        std::string data, std::string salt)
+    void dht_put_mutable_item(lt::session& ses, bytes private_key, bytes public_key,
+        bytes data, bytes salt)
     {
         TORRENT_ASSERT(private_key.size() == 64);
         TORRENT_ASSERT(public_key.size() == 32);
```

Comment:
these asserts should probably be run-time checks

Reply:
Did that in #6355